### ISSUE TITLE
Accessibility Issue: Videos lack captions

### ISSUE THREAD

**Issue by corderophilosophy** [FreeCodeCamp/FreeCodeCamp#7042]
There should be captions in the videos. Even if there isn't any more information in the videos than is contained in the Instructions, someone who needs the captions would have to do the extra work to find that out. That takes time away from the process of learning.

**Comment by BerkeleyTrue** [comment]
closed in favor of https://github.com/FreeCodeCamp/hikes/issues/160
